### ISSUE TITLE
Popup window probably outdated

### ISSUE THREAD

**Issue by Hideman85** [Hunter5000/twitch-fox#15]
![image](https://user-images.githubusercontent.com/16165119/93375125-fb4a1180-f857-11ea-97df-c228aeb89aa6.png)

**Comment by Rex000** [comment]
Yes, I'm having the same issue. One more nail in the coffin I guess.
